pontoon/sync/migrations/0002_change_pontoon_sync_email.py

```diff
@@ -8,13 +8,13 @@
 def update_sync_user_email(apps, schema_editor):
     User = apps.get_model("auth", "User")
     User.objects.filter(email="pontoon-sync@mozilla.org").update(
-        email="pontoon-sync@mozilla.com"
+        email="pontoon-sync@example.com"
```

Comment:
Same as above. Please revert the changes to the migration files and add a new data migration in the `sync` app.